@xen-orchestra/lite/src/components/pool/dashboard/PoolDashboardStorageUsage.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <UiCard>
+    <UiTitle type="h4">{{ $t("storage-usage") }}</UiTitle>
+    <UsageBar :data="data.result" :nItems="5">
+      <template #header>
+        <span>{{ $t("storage") }}</span>
+        <span>{{ $t("top-5") }}</span>
+      </template>
+      <template #footer v-if="showFooter">
+        <div class="footer-card">
+          <p>{{ $t("total-used") }}:</p>
+          <div class="footer-value">
+            <p>{{ percentUsed }}%</p>
+            <p>
+              {{ formatSize(data.usedSize) }}
+            </p>
+          </div>
+        </div>
+        <div class="footer-card">
+          <p>{{ $t("total-free") }}:</p>
+          <div class="footer-value">
+            <p>{{ 100 - percentUsed }}%</p>
```

Comment:
We should avoid `100 - num`, since it can cause display issue.

For example `100 - 20.04` would display `79.96000000000001`
```suggestion
            <p>{{ percentFree }}%</p>
```